zerver/models/custom_profile_fields.py

```diff
@@ -69,7 +69,9 @@ class CustomProfileField(models.Model):
 
     realm = models.ForeignKey(Realm, on_delete=CASCADE)
     name = models.CharField(max_length=NAME_MAX_LENGTH)
+    rendered_name = models.TextField(null=True, default=None)
     hint = models.CharField(max_length=HINT_MAX_LENGTH, default="")
+    rendered_hint = models.TextField(null=True, default=None)
```

Comment:
Can we just set these fields for existing `CustomProfileField` objects as well? We did so when adding `rendered_description` field for streams in 73d26c8b28.

Reply:
I updated it to use `default=""` for existing objects. However, while fetching the initial state, I am popping the `rendered_name` and `rendered_hint` fields if `rendered_name == ""` (indicating for existing objects). Do you think this is the right way to do it, or do I handle it at the frontend?